HelpSource/Reference/Functions.schelp

```diff
@@ -87,18 +89,57 @@ code::
 
 See link::Reference/Literals:: for more information.
 
+
+
+section:: Variable Arguments
+
+What happens if you don't know how many arguments a function should have?
```

Comment:
For reference (unlike guides) it would be good not to address th ereader directly. At least this is the style that exists and I'd prefer not to mix too much.